app/graph/menu_type.rb

```diff
@@ -0,0 +1,10 @@
+MenuType = GraphQL::ObjectType.define do
+  name 'Menu'
+  description 'Some lunch menu you can rate'
+  # `!` marks a field as "non-null"
+  field :id, !types.ID
+  field :title, !types.String
+  field :price, !types.Float
+  field :description, !types.String
+  field :ratings, types[RatingType]
```

Comment:
Wouldn't an empty list also be a non-null value? If so, we could make this field non-null as well.